src/patterns/validation/index.md.njk

```diff
@@ -0,0 +1,92 @@
+---
+title: Recover from errors (validation)
+description: Check the answers users give to make sure they’re valid - and if there’s an error, tell them what's wrong and how to fix it
+section: Patterns
+theme: Help users to…
+aliases: 
+backlog_issue_id: 
+layout: layout-pane.njk
+---
+
+Check the information the user gives you to make sure it’s valid. If there's a validation error, tell the user what's gone wrong and how to fix it. Turn off HTML5 validation.
+
+## When to use this pattern
+
+Validation should be a last resort. Minimise your chances of needing to show an error message by:
```

Comment:
> Validation should always be a last resort

I find this odd phrasing. Services should try to do the hard work so it's less common - but validation is expected on the majority of service pages. Of the services I previously assessed, validation was often forgotten about / not given enough attention.

Reply:
Thank yes, we've updated the language here.